lib/rules/no-unstable-nested-components.js

```diff
@@ -276,12 +276,6 @@ module.exports = {
     schema: [{
       type: 'object',
       properties: {
-        customValidators: {
```

Comment:
This option appears to be unused in this rule. eslint-doc-generator caught this because it checks that all rule options are mentioned in the rule doc.

Reply:
It's used by the propTypes util, so even if it's missing documentation, it's still used.

The docs are *never* the source of truth, only the code is, so that "catch" seems a bit backwards.

Reply:
I also checked the tests for this rule and didn't see the option tested. That seems like an issue. I'll add the option back.

Reply:
Indeed, missing tests is far more worrisome than missing docs, but either way removing the option could break people, so we'll need to add the missing tests/docs :-) thanks for noticing!

Reply:
Added the option back and mentioned it in the docs. Someone should follow-up separately to add the missing test.